doc/nrf/gs_installing.rst

```diff
@@ -219,6 +219,51 @@ Therefore, remember to regularly check for updates:
          pip3 install -U west
 
 
+Installing additional toolchains for Connected Home over IP applications development
+====================================================================================
+
+.. note::
+   If you are not interested in the `Connected Home over IP`_ applications, you can ommit steps presented in this section.
+
+`Connected Home over IP`_ project is using `GN`_ meta-build system to generate files for `Ninja`_ and both tools are necessary in order to build an applications.
+However Ninja is also required tool for Zephyr, so it is assumed that it should be installed with the rest of Zephyr's dependencies and GN is the only missing |NCS| requirement.
+
+.. note::
+   Connected Home over IP project supports only Linux and macOS platforms. Windows platform is not available.
+
+Download GN binary archive by entering the following commands:
+
+.. tabs::
+
+   .. group-tab:: Linux
+
+      .. parsed-literal::
+         :class: highlight
+
+         wget -O gn_archive https://chrome-infra-packages.appspot.com/dl/gn/gn/linux-amd64/+/latest
```

Comment:
The commands will create `gn_archive` and `gn` files in the current directory which is unknown and which a user may not have write permissions to. Should we rather recommend using a known location? Also gn_archive should be deleted once no longer needed. E.g.

```
cd ~
wget -O gn.zip https://chrome-infra-packages.appspot.com/dl/gn/gn/linux-amd64/+/latest
unzip gn.zip gn
rm gn.zip
```